data/saas/config/domo_config.yml

```diff
@@ -0,0 +1,75 @@
+saas_config:
+  fides_key: domo_connector_example
+  name: Domo SaaS Config
+  description: A sample schema representing the Domo connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: client_id
+    - name: client_secret
+    - name: email
+
+  client_config:
+    protocol: https
+    host:
+      connector_param: domain
+    authentication:
+      strategy: bearer_authentication
+      configuration:
+        token:
+          connector_param: access_token
+          # TODO - metaEndpoints wasn't supported at the time it was written
```

Comment:
We didn't copy over the concept of meta endpoints to Fidesops. We can implement this as a custom auth strategy that is configured to call the refresh endpoint before each request (no actions needed for now, that is to be done as part of a separate ticket).